<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modification_window</class>
 <widget class="QWidget" name="modification_window">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>830</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(239, 239, 239)</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="_my_return_button">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
color:rgb(100, 100, 100);
padding
-left: 20px;
</string>
          </property>
          <property name="text">
           <string>&lt; RETOUR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_my_picture_label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-right: 140px;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#838383;&quot;&gt;Photo du XX Mois 20XX à XX:XX&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; color:#a7a7a7;&quot;&gt;XXX Mo - Format XXX - Image originale&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(167, 167, 167);
border:none;

</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="_my_recadrer">
          <property name="styleSheet">
           <string notr="true">border-style: solid;
border-radius: 5px;
border-width: 1px;
border-color: grey;
width : 90px;
height :20px;
background-color : rgb(224, 224, 224);
color:rgb(143, 143, 143);
font-size:15px;
</string>
          </property>
          <property name="text">
           <string>Recadrer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="_my_redimensionner">
          <property name="styleSheet">
           <string notr="true">border-style: solid;
border-radius: 5px;
border-width: 1px;
border-color: grey;
width : 90px;
height :20;
background-color : rgb(224, 224, 224);
color:rgb(143, 143, 143);
font-size:15px;</string>
          </property>
          <property name="text">
           <string>Redimensionner</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="_my_reset">
          <property name="styleSheet">
           <string notr="true">border-style: solid;
border-radius: 5px;
border-width: 1px;
border-color: grey;
width : 90px;
height :20;
background-color : rgb(224, 224, 224);
color:rgb(143, 143, 143);
font-size:15px;</string>
          </property>
          <property name="text">
           <string>Revenir à l'original</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="_my_detail_box">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>8000</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(250, 250, 250);
border-right: 1px black solid;

</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="_my_dominant_color_label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#8b8b8b;&quot;&gt;Couleur dominante&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_my_color_dominant_edit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(180, 179, 179);
border: 1px solid rgb(180, 179, 179);
border-radius:5px;


</string>
          </property>
          <property name="text">
           <string>#F8F7F7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_my_note_label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#8b8b8b;&quot;&gt;Note&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_my_rating_layout">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="_my_commentaire_label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#8b8b8b;&quot;&gt;Commentaire&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_my_commentaire_edit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>75</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(180, 179, 179);
border: 1px solid rgb(180, 179, 179);
border-radius:5px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Vous pouvez écrire un petit commentaire en cliquant ici</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="_my_descriptif_label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#8b8b8b;&quot;&gt;Descriptif&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; color:#8b8b8b;&quot;&gt;Ajouter un mot clé descriptif ...&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_my_add_tag_desc_button">
            <property name="styleSheet">
             <string notr="true">color:rgb(180, 179, 179);
border: 1px solid rgb(180, 179, 179);
border-radius:5px;
</string>
            </property>
            <property name="text">
             <string>ajouter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="_my_descriptif_tag_area">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="_desc_area_content">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>472</width>
             <height>98</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="_my_ressenti_label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#8b8b8b;&quot;&gt;Ressenti&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; color:#8b8b8b;&quot;&gt;Ajouter un mot clé descriptif ...&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_my_add_tag_ress_button">
            <property name="styleSheet">
             <string notr="true">color:rgb(180, 179, 179);
border: 1px solid rgb(180, 179, 179);
border-radius:5px;

</string>
            </property>
            <property name="text">
             <string>ajouter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="_my_ressenti_tag_area">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="_feeling_area_content">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>472</width>
             <height>98</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="_my_catgorie_label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#8b8b8b;&quot;&gt;Catégorie&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; color:#8b8b8b;&quot;&gt;Ajouter un mot clé de catégorie ...&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_my_add_tag_cat_button">
            <property name="styleSheet">
             <string notr="true">color:rgb(180, 179, 179);
border: 1px solid rgb(180, 179, 179);
border-radius:5px;
</string>
            </property>
            <property name="text">
             <string>ajouter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="_my_categorie_tag_area">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="_cat_area_content">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>472</width>
             <height>98</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <widget class="QWidget" name="gridLayoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>471</width>
              <height>101</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="_cat_area"/>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="_my_picture">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>3000</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:0px;</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="_my_picture_content">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>776</width>
            <height>711</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="zoom_layout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-left:200%</string>
           </property>
           <property name="text">
            <string>Zoom:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="_my_slider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="_my_download_button">
           <property name="text">
            <string>Enregistrer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_my_redimensionner</tabstop>
  <tabstop>_my_reset</tabstop>
  <tabstop>_my_color_dominant_edit</tabstop>
  <tabstop>_my_commentaire_edit</tabstop>
  <tabstop>_my_add_tag_desc_button</tabstop>
  <tabstop>_my_picture</tabstop>
  <tabstop>_my_descriptif_tag_area</tabstop>
  <tabstop>_my_add_tag_ress_button</tabstop>
  <tabstop>_my_ressenti_tag_area</tabstop>
  <tabstop>_my_add_tag_cat_button</tabstop>
  <tabstop>_my_categorie_tag_area</tabstop>
  <tabstop>_my_recadrer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
